boot.sh

```diff
@@ -14,7 +14,7 @@ then
     diff -rq /config /config.original > /dev/null
     CONFIG_FILES_CHANGED="$?"
 
-    if [ ! -f /usr/src/build/app.js ]
+    if [ ! -f /usr/src/dist/app.js ]
```

Comment:
i rewired the old `build` directory to `dist` and kept the `build` directory for the combined sources of template and service. Apparently some IDEs need both to keep their breakpoints in a working state...